guest/handle/sharedlister/api/sharedlister.go

```diff
@@ -0,0 +1,15 @@
+package api
+
+import (
+	guestapi "sigs.k8s.io/kube-scheduler-wasm-extension/guest/api"
+)
+
+type SharedLister interface {
+	NodeInfos() guestapi.NodeInfoList
```

Comment:
minor:
How about adding the comment that StorageInfos are not implemented yet?

Reply:
Ditto, I don't want to leave comments on everything unsupported.